app/experimenter/visualization/api/v1/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from experimenter.visualization.api.v1.views import VisualizationView
+
+urlpatterns = [
+    url(
+        r"^(?P<slug>[\w-]+)/$",
+        VisualizationView.as_view(),
+        name="visualization",
```

Comment:
Usually we give views names like `app-name` so since you already called this app `visualization` maybe you can find a word to use to describe what this specific view is doing?